bosh/releases/pre_render_scripts/api/routing-api/bpm/patch_bpm.sh

```diff
@@ -5,15 +5,18 @@ set -o errexit -o nounset
 target="/var/vcap/all-releases/jobs-src/routing/routing-api/templates/bpm.yml.erb"
 
 PATCH=$(cat <<'EOT'
-@@ -11,7 +11,7 @@
+@@ -11,8 +11,10 @@
      - -timeFormat
      - rfc3339
      - -ip
 -    - <%= spec.ip %>
-+    - 0.0.0.0
++    - "$(POD_IP)"
```

Comment:
Huh, I wonder what syntax this is. Seems to work, though:
> `/var/vcap/packages/routing-api/bin/routing-api -config /var/vcap/jobs/routing-api/config/routing-api.yml -port 3000 -logLevel info -timeFormat rfc3339 -ip 172.17.0.23`

Reply:
https://kubernetes.io/docs/tasks/inject-data-application/define-command-argument-container/#use-environment-variables-to-define-arguments

k8s sorcery!